### PR TITLE
CW-511-Tablet-iPad-keyboard-issue

### PR DESCRIPTION
**Pull request by Serhii-Borodenko**
Issue Number (if Applicable): Fixes #

# Description

Please include a summary of the changes and which issue is fixed / feature is added. 

# Pull Request - Checklist  

- [ ] Initial Manual Tests Passed
- [ ] Double check modified code and verify it with the feature/task requirements
- [ ] Format code
- [ ] Look for code duplication
- [ ] Clear naming for variables and methods

### CODE REVIEW

File: lib/src/screens/pin_code/pin_code_widget.dart [hunk 1]
```diff
@@ -191,7 +191,7 @@ class PinCodeState<T extends PinCodeWidget> extends State<T> {
               child: Center(
                 child: ConstrainedBox(
                   constraints: BoxConstraints(
-                    maxWidth: ResponsiveLayoutUtil.kDesktopMaxWidthConstraint,
+                    maxWidth: ResponsiveLayoutUtilBase.kDesktopMaxDashBoardWidthConstraint,
```
Comment (OmarHatem28) [inline lib/src/screens/pin_code/pin_code_widget.dart:194]: is this change intended?

Comment (Serhii-Borodenko) [reply lib/src/screens/pin_code/pin_code_widget.dart]: No, it was a mistake, fixed, thank you.

**Review by OmarHatem28 — APPROVED** [review]
